app/views/govuk_publishing_components/components/layout_header/_navigation_items.html.erb

```diff
@@ -1,3 +1,5 @@
+<% navigation_aria_label ||= "Top level" %>
```

Comment:
I'm obviously late to this party, but it would be great if this text could be pulled from the translations file.